analyzing_text.py

```diff
@@ -0,0 +1,65 @@
+from gathering_text import get_trump_tweets, get_clinton_tweets
+import re
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import doctest
+analyzer = SentimentIntensityAnalyzer()
+
+
+def print_word_freqs(texts):
+    '''Prints the frequency of words used in the list texts'''
+    texts = [s + ' ' for s in texts]
+    str_copy = str(texts)
```

Comment:
Converting to a string then back to a list seems a bit inefficient. There are some list functions to clean data nicely, or you could import a package like pandas to clean your dataset.